Failed to build on Ubuntu 20.04 LTS

Hi,
  While trying to build release 2.9.3 on Ubuntu 20.04 LTS, I got the following error
````
Making install in data
make[1]: Entering directory '/home/ubuntu/x/udisks-2.9.3/data'
Making install in tmpfiles.d
make[2]: Entering directory '/home/ubuntu/x/udisks-2.9.3/data/tmpfiles.d'
make[3]: Entering directory '/home/ubuntu/x/udisks-2.9.3/data/tmpfiles.d'
make[3]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/usr/lib/tmpfiles.d'
 /usr/bin/install -c -m 644 udisks2.conf '/usr/lib/tmpfiles.d'
make[3]: Leaving directory '/home/ubuntu/x/udisks-2.9.3/data/tmpfiles.d'
make[2]: Leaving directory '/home/ubuntu/x/udisks-2.9.3/data/tmpfiles.d'
make[2]: Entering directory '/home/ubuntu/x/udisks-2.9.3/data'
  GEN      org.freedesktop.UDisks2.policy
make[3]: Entering directory '/home/ubuntu/x/udisks-2.9.3/data'
make[3]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/usr/local/share/bash-completion/completions'
 /usr/bin/install -c -m 644 completions/udisksctl '/usr/local/share/bash-completion/completions'
 /usr/bin/mkdir -p '/usr/local/share/dbus-1/system.d'
 /usr/bin/install -c -m 644 org.freedesktop.UDisks2.conf '/usr/local/share/dbus-1/system.d'
 /usr/bin/mkdir -p '/usr/local/share/dbus-1/system-services'
 /usr/bin/install -c -m 644 org.freedesktop.UDisks2.service '/usr/local/share/dbus-1/system-services'
  GEN      org.freedesktop.UDisks2.policy
 /usr/bin/mkdir -p '/usr/local/share/polkit-1/actions'
 /usr/bin/install -c -m 644 ./org.freedesktop.UDisks2.policy '/usr/local/share/polkit-1/actions'
/usr/bin/install: cannot stat './org.freedesktop.UDisks2.policy': No such file or directory
make[3]: *** [Makefile:563: install-polkitDATA] Error 1
make[3]: Leaving directory '/home/ubuntu/x/udisks-2.9.3/data'
make[2]: *** [Makefile:796: install-am] Error 2
make[2]: Leaving directory '/home/ubuntu/x/udisks-2.9.3/data'
make[1]: *** [Makefile:632: install-recursive] Error 1
make[1]: Leaving directory '/home/ubuntu/x/udisks-2.9.3/data'
make: *** [Makefile:514: install-recursive] Error 1
````

The issue is that `org.freedesktop.UDisks2.policy` isn't actually generated, even if `make` output seems to indicate it is.
If I manually generate it by copy (note there are no template variables):
````
$ cp data/org.freedesktop.UDisks2.policy.in data/org.freedesktop.UDisks2.policy
````
I'm able to get past the error and build successfully.

I just had the same issue. My output for `make V=1 install` was as follows

```
Making install in data
make[1]: Entering directory '/home/rgingras/Downloads/udisks-2.9.4/data'
Making install in tmpfiles.d
make[2]: Entering directory '/home/rgingras/Downloads/udisks-2.9.4/data/tmpfiles.d'
make[3]: Entering directory '/home/rgingras/Downloads/udisks-2.9.4/data/tmpfiles.d'
make[3]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/usr/lib/tmpfiles.d'
 /usr/bin/install -c -m 644 udisks2.conf '/usr/lib/tmpfiles.d'
make[3]: Leaving directory '/home/rgingras/Downloads/udisks-2.9.4/data/tmpfiles.d'
make[2]: Leaving directory '/home/rgingras/Downloads/udisks-2.9.4/data/tmpfiles.d'
make[2]: Entering directory '/home/rgingras/Downloads/udisks-2.9.4/data'
: --xml --template org.freedesktop.UDisks2.policy.in -d ../po -o org.freedesktop.UDisks2.policy
make[3]: Entering directory '/home/rgingras/Downloads/udisks-2.9.4/data'
make[3]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/usr/local/share/bash-completion/completions'
 /usr/bin/install -c -m 644 completions/udisksctl '/usr/local/share/bash-completion/completions'
 /usr/bin/mkdir -p '/usr/local/share/dbus-1/system.d'
 /usr/bin/install -c -m 644 org.freedesktop.UDisks2.conf '/usr/local/share/dbus-1/system.d'
 /usr/bin/mkdir -p '/usr/local/share/dbus-1/system-services'
 /usr/bin/install -c -m 644 org.freedesktop.UDisks2.service '/usr/local/share/dbus-1/system-services'
: --xml --template org.freedesktop.UDisks2.policy.in -d ../po -o org.freedesktop.UDisks2.policy
 /usr/bin/mkdir -p '/usr/local/share/polkit-1/actions'
 /usr/bin/install -c -m 644 ./org.freedesktop.UDisks2.policy '/usr/local/share/polkit-1/actions'
/usr/bin/install: cannot stat './org.freedesktop.UDisks2.policy': No such file or directory
make[3]: *** [Makefile:563: install-polkitDATA] Error 1
make[3]: Leaving directory '/home/rgingras/Downloads/udisks-2.9.4/data'
make[2]: *** [Makefile:796: install-am] Error 2
make[2]: Leaving directory '/home/rgingras/Downloads/udisks-2.9.4/data'
make[1]: *** [Makefile:632: install-recursive] Error 1
make[1]: Leaving directory '/home/rgingras/Downloads/udisks-2.9.4/data'
make: *** [Makefile:514: install-recursive] Error 1
```

I noticed that `GEN` was expanding to the `:` instead of `/usr/bin/msgfmt` and was able to successfully install after getting the gettext package which provides msgfmt on Ubuntu.

--- a/configure.ac
+++ b/configure.ac
@@ -593,6 +593,11 @@ GETTEXT_PACKAGE=udisks2
 AC_SUBST([GETTEXT_PACKAGE])
 AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE],["$GETTEXT_PACKAGE"],[gettext domain])
 
+# explicit check for msgfmt, needed for polkit files processing
+AS_IF([test -z "$MSGFMT" -o "$MSGFMT" = ":"], [
+  AC_MSG_ERROR([msgfmt not found (gettext package)])
+])
+
 AC_DEFINE([PROJECT_SYSCONF_DIR], ["udisks2"], [Project configuration directory])
 AC_DEFINE([PACKAGE_NAME_UDISKS2], ["udisks2"], [Full package name for UDisks2 compatibility])
 

